src/main/java/com/employeesService/EmployeesService/model/DepartmentEmployeeDTO.java

```diff
@@ -0,0 +1,33 @@
+package com.employeesService.EmployeesService.model;
+
+import javax.persistence.JoinColumn;
+import java.io.Serializable;
+import java.time.LocalDate;
+
+public class DepartmentEmployeeDTO implements Serializable {
```

Comment:
I don't think it's needed
```suggestion
public class DepartmentEmployeeDTO {
```